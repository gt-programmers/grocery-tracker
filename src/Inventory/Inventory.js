import React from 'react';
import './Inventory.scss';

const Inventory = () => {
  return (
    <section className="inventory page">
      <p>Inventory</p>
    </section>
  )
}

export default Inventory;