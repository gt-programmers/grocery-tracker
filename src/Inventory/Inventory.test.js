import React from 'react';
import Inventory from './Inventory';
import { shallow } from 'enzyme';

describe("Inventory", () => {
  let wrapper = shallow( <Inventory />)


  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
})