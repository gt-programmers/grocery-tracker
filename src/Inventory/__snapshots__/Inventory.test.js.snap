// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Inventory should match the snapshot 1`] = `
<section
  className="inventory page"
>
  <p>
    Inventory
  </p>
</section>
`;
